src/jmc/compile/datapack_data.py

```diff
@@ -6,6 +6,91 @@
 if TYPE_CHECKING:
     from ..compile.tokenizer import Token
 
+@dataclass
+class MinecraftAdvancements:
```

Comment:
This is not what I meant you should define in `jmc_command.py` outside of class

`MINECRAFT_ADVANCEMENTS = {`